pkg/webui/console/components/events/previews/shared/description-list/description-list.styl

```
@@ -37,6 +37,7 @@
     margin-left: 0
 
 .value
+  color: var(--text-neutral)
```

```suggestion
  color: var(--c-text-neutral)
```
Aren't the colors prefixed with `c`?